sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/models/TelemetryItem.java

```diff
@@ -203,82 +180,28 @@ public TelemetryItem setSequence(String sequence) {
     }
 
     /**
-     * Get the instrumentationKey property: The instrumentation key of the Application Insights
-     * resource.
+     * Get the instrumentationKey property: The instrumentation key of the Application Insights resource.
      *
      * @return the instrumentationKey value.
      */
     public String getInstrumentationKey() {
         return this.instrumentationKey;
     }
 
-    @JsonIgnore
-    public String getConnectionString() {
-        return connectionString;
-    }
-
-    @JsonIgnore
-    public TelemetryItem setConnectionString(String connectionString) {
-        this.connectionString = connectionString;
-        this.instrumentationKey = ConnectionString.parse(connectionString).getInstrumentationKey();
-        return this;
-    }
-
-    @JsonIgnore
-    public TelemetryItem setConnectionString(ConnectionString connectionString) {
-        this.connectionString = connectionString.getOriginalString();
-        this.instrumentationKey = connectionString.getInstrumentationKey();
-        return this;
-    }
-
-    @JsonIgnore
-    public TelemetryItem setConnectionString(StatsbeatConnectionString connectionString) {
-        instrumentationKey = connectionString.getInstrumentationKey();
-        // TODO (heya) turn StatsbeatConnectionString into a real connection string?
-        this.connectionString =
-            "InstrumentationKey="
-                + instrumentationKey
-                + ";IngestionEndpoint="
-                + connectionString.getIngestionEndpoint();
```

Comment:
@trask  these are deleted by the new models. connection string is in the json. i guess that is why. we don't need connection string in the data model. we do have it in the telemetryclient. i think the new model is correct, can you pls confirm?